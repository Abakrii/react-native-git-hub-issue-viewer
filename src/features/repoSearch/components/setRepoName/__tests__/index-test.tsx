import React from 'react';
import renderer from 'react-test-renderer';
import SetRepoName from '../';

test('renders correctly', () => {
  const tree = renderer.create(<SetRepoName />).toJSON();
  expect(tree).toMatchSnapshot();
});
