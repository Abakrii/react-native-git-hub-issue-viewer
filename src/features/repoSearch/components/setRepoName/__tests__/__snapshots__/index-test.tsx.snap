// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<TextInput
  allowFontScaling={true}
  onChangeText={[Function]}
  placeholder="Paste a link to a GitHub repo!"
  rejectResponderTermination={true}
  style={
    Object {
      "borderBottomColor": "#DDD",
      "borderBottomWidth": 1,
      "marginBottom": 10,
      "paddingHorizontal": 8,
      "paddingVertical": 6,
    }
  }
  underlineColorAndroid="transparent"
/>
`;
